WPMedia/fusion-news-theme-blocks
# TMEDIA 722 Links bar cleanup

## Description

Update links bar to use v2 architecture

## Jira Ticket

- [TMEDIA-722](https://arcpublishing.atlassian.net/browse/TMEDIA-722)

## Acceptance Criteria

Convert the links bar block to v2 architecture

Links Bar block is migrated to the new architecture

Includes Custom fields of:

‘Navigation’ - dropdown with option for ‘navigation-hierarchy'

‘Content Source' -dropdown with option for ‘site-service hierarchy’

‘hierarchy’ - default is an empty field

‘sectionId' - default is an empty field

Includes Configure Content fields of:

‘Toggle light mode’ - defaulted to false

‘Arial-label' - open field with default of “Breaking News Alert” and existing help text

## Test Steps

1. Checkout this branch `git checkout TMEDIA-722-links-bar-cleanup`
2. Use the associated pr for this feature pack `TMEDIA-722-links-bar-cleanup`
3. Make sure you're on the themes internal custom sandbox .env data.
4. Run fusion repo with linked blocks `npx fusion start -f -l @wpmedia/links-bar-block`
5. Create a page with this block 
![Screen Shot 2022-06-13 at 11 28 04](https://user-images.githubusercontent.com/5950956/173400994-95841998-781a-4a7e-a2f5-b9918182bea8.png)
6. Add the block 
7. Update the content with default values 
![Screen Shot 2022-06-13 at 11 28 26](https://user-images.githubusercontent.com/5950956/173401045-0f9255fe-a3f7-4fb9-bdbf-18f6b09c1deb.png)
8. See the separator and the links


## Effect Of Changes

### Before

- showed links, separator, and divider

### After

- showed links, separator, and divider. but uses v2 naming and styles

## Dependencies or Side Effects

- manifest https://github.com/WPMedia/arc-themes-manifests/pull/67 https://github.com/WPMedia/arc-themes-manifests/pull/71
- feature pack https://github.com/WPMedia/arc-themes-blocks/pull/1378

## Author Checklist

        please explain why (so that we can fix it whenever it gets refactored).
- [x] Confirmed relevant documentation has been updated/added. -> removing placeholder, updating information

## Reviewer Checklist

_The reviewer of the PR should copy-paste this template into the review comments on review._
